docs/src/content/components/dropdown.mdx

```diff
@@ -359,14 +343,52 @@ function Example() {
 export default Example
 ```
 
-## Usage & Types
+### Menu scrolling
 
-<Usage
-  install="npm install @pluralsight/ps-design-system-dropdown"
-  import="import Dropdown from '@pluralsight/ps-design-system-dropdown'"
-  packageName="dropdown"
-  version={props.version}
-/>
+The dropdown component menu has a max height of 400px and will scroll for content needing more vertical space.
+
+```typescript
+import React from 'react'
+import Dropdown from '@pluralsight/ps-design-system-dropdown'
+
+const Example = () => (
+  <Dropdown
+    label="Max height example"
+    menu={
+      <>
+        <Dropdown.Item>One item</Dropdown.Item>
+        <Dropdown.Item>Two item</Dropdown.Item>
+        <Dropdown.Item>Three item</Dropdown.Item>
+        <Dropdown.Item>Four item</Dropdown.Item>
+        <Dropdown.Item>Five item</Dropdown.Item>
+        <Dropdown.Item>Six item</Dropdown.Item>
+        <Dropdown.Item>Seven item</Dropdown.Item>
+        <Dropdown.Item>Eight item</Dropdown.Item>
+        <Dropdown.Item>Nine item</Dropdown.Item>
+        <Dropdown.Item>Ten item</Dropdown.Item>
+        <Dropdown.Item>Eleven item</Dropdown.Item>
+        <Dropdown.Item>Twelve item</Dropdown.Item>
+        <Dropdown.Item>Thirteen item</Dropdown.Item>
+        <Dropdown.Item>Fourteen item</Dropdown.Item>
+        <Dropdown.Item>Fifteen item</Dropdown.Item>
+      </>
+    }
+  />
+)
+
+export default Example
+```
+
+## Accessibility
+
+**WCAG 2.1 AA Partial Compliance**
```

Comment:
nbd, but I'd do something like this for these options:

- **Partially WCAG 2.1 AA Compliant**
- **WCAG 2.1 AA Compliant**

Why?

- Reads more like a status badge, applied to the component
- More easily scannable with "partial" at the front and looking different than full compliance

Reply:
👍